demos/python_demos/instance_segmentation_demo/README.md

```diff
@@ -9,23 +9,23 @@ This demo demonstrates how to run Instance Segmentation models from `Detectron`
 The demo application expects an instance segmentation model in the Intermediate Representation (IR) format with the following constraints:
 
 * Two inputs: `im_data` for input image and `im_info` for meta-information about the image (actual height, width and scale).
-* At least four outputs containing:   
-    * `boxes` with absolute bounding box coordinates of the input image   
-    * `scores` with confidence scores for all bounding boxes  
-    * `classes` with object class IDs for all bounding boxes   
+* At least four outputs including:
+    * `boxes` with absolute bounding box coordinates of the input image
+    * `scores` with confidence scores for all bounding boxes
+    * `classes` with object class IDs for all bounding boxes
     * `raw_masks` with fixed-size segmentation heat maps for all classes of all bounding boxes
 
 As input, the demo application takes:
-* a path to a single image file or a folder with image files specified with a command line argument `--images`
-* a path to a video file or a numeric ID of a web-camera specified with a command line argument `--video`
+* a path to a single image file, a video file or a numeric ID of a web camera specified with a command-line argument `-i`
 
 The demo workflow is the following:
 
-1. The demo application reads images/video frames one by one, resizes them to fit into the input image blob of the network (`im_data`) keeping the original image aspect ratio.
+1. The demo application reads image/video frames one by one, resizes them to fit into the input image blob of the network (`im_data`).
 2. The `im_info` input blob passes resulting resolution and scale of a pre-processed image to the network to perform inference.
 3. The demo visualizes the resulting instance segmentation masks. Certain command-line options affect the visualization:
     * If you specify `--show_boxes` and `--show_scores` arguments, bounding boxes and confidence scores are also shown.
-    * If you specify video as a source of images with the `--video` option, the demo shows the same object instance with the same color throughout the whole video using simple tracking. It assumes more or less static scene with instances in two frames being a part of the same track if intersection over union of the masks is greater than the 0.5 threshold.
+    * By default, tracking is used to show object instance with the same color throughout the whole video.
+      It assumes more or less static scene with instances in two frames being a part of the same track if intersection over union of the masks is greater than the 0.5 threshold. To disable tracking, specify `--no_track` argument.
```

Comment:
`` specify `--no_track` argument.` `--> `` specify the `--no_track` argument.``

Reply:
Fixed.